docs/outlook/outlook-mobile-addins.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Outlook add-ins for Outlook Mobile
-description: Outlook mobile add-ins are supported on all Office 365 Commercial accounts, Outlook.com accounts, and support is coming soon to gmail accounts.
+description: Outlook mobile add-ins are supported on all Microsoft 365 business accounts, Outlook.com accounts, and support is coming soon to gmail accounts.
```

Comment:
Just curious but how do we know that Commercial translates to business (vs enterprise, for e.g.)?

Reply:
There are so many different subscriptions, including within Microsoft 365 for business. It's best to keep it generic, using "Microsoft 365 for business" https://www.microsoft.com/en-us/microsoft-365/business